app/qml/map/MapWrapper.qml

```diff
@@ -274,6 +294,8 @@ Item {
           root.nothingIdentified()
         }
       }
+      if ( root.state === "navigation" )
+        exitNavigationClicked();
```

Comment:
I wonder whether we want this at all. What do you think @NEDJIMAbelgacem? I am afraid that users might accidentally tap on their screen and thus lose the navigation.. When I look at other apps, they do not seem to have such "feature". I would leave the **X** button an only way how to exit navigation :)

Reply:
I was thinking that it is intuitive for me to minimize panels when the map is clicked. But after thinking about it for a bit I think navigation shouldn't be stopped because you clicked on the map (I remembered Google maps navigation).